product-scenarios/1-manage-users-roles/1.1-user-registration/1.1.2-user-registration-by-admin/1.1.2.1-user-registration-through-apis/1.1.2.1.2-scim-2.0/src/test/java/org/wso2/identity/scenarios/test/scim2/SCIMConstants.java

```diff
@@ -20,32 +20,137 @@
 
 public class SCIMConstants {
 
-    public static final String SCIM2_USERS_ENDPOINT = "/scim2/Users";
-    public static final String SCIM2_BULK_USERS_ENDPOINT = "/scim2/Bulk";
-    public static final String SCHEMAS_ATTRIBUTE = "schemas";
-    public static final String GIVEN_NAME_ATTRIBUTE = "givenName";
-    public static final String FAMILY_NAME_ATTRIBUTE = "familyName";
-    public static final String EMAILS_ATTRIBUTE = "emails";
-    public static final String EMAIL_TYPE_WORK_ATTRIBUTE = "work";
-    public static final String EMAIL_TYPE_HOME_ATTRIBUTE = "home";
-    public static final String TYPE_PARAM = "type";
-    public static final String VALUE_PARAM = "value";
-    public static final String PRIMARY_PARAM = "primary";
-    public static final String NAME_ATTRIBUTE = "name";
-    public static final String USER_NAME_ATTRIBUTE = "userName";
-    public static final String PASSWORD_ATTRIBUTE = "password";
-    public static final String ID_ATTRIBUTE = "id";
-    public static final String FAMILY_NAME_CLAIM_VALUE = "scim2";
-    public static final String USERNAME = "scim2user";
-    public static final String PASSWORD = "scim2pwd";
-    public static final String CONTENT_TYPE_APPLICATION_JSON = "application/json";
-    public static final String GIVEN_NAME_CLAIM_VALUE = "user1";
-    public static final String SCIM2USER = "scim2user";
-    public static final String SCIM2PASSWORD = "scim2pwd";
-    public static final String DISPLAY_NAME = "displayName";
-    public static final String DISPLAY = "display";
-    public static final String MEMBERS = "members";
-    public static final String VALUE = "value";
-    public static final String ROLE_NAME = "TestRole";
-    public static final String ERROR_SCHEMA = "urn:ietf:params:scim:api:messages:2.0:Error";
+	   public static final String SCIM2_USERS_ENDPOINT = "scim2";
```

Comment:
Seems to be an unwanted indentation ?

Reply:
thanks, corrected by removing the indentation